PoGo.NecroBot.Logic/Tasks/CatchNearbyPokemonsTask.cs

```diff
@@ -29,6 +29,12 @@ public static void Execute(Context ctx, StateMachine machine)
                     continue;
                 }
 
+                if (ctx.Inventory.GetPokemons().Result.Any(p => p.PokemonId == pokemon.PokemonId) && ctx.LogicSettings.DontCatchDuplicatedPokemon)
+                {
+                    Logger.Write("Already Have " + pokemon.PokemonId + " Skipped");
```

Comment:
there is already a logger call before mine 
https://github.com/plsfixrito/NecroBot/blob/790a55f3e7997833b414acb7da80accc45b6a4bf/PoGo.NecroBot.Logic/Tasks/CatchNearbyPokemonsTask.cs#L25-L30